test/e2e/helpers.js

```diff
@@ -408,6 +409,7 @@ const importSRPOnboardingFlow = async (driver, seedPhrase, password) => {
   await driver.assertElementNotPresent('.loading-overlay');
 };
 
+// This function will be deprecated, please use page object functions in onboarding.flow.ts and in pages/onboarding/*
```

Comment:
Maybe we could add it in jsdoc to have it indicated in editor when invoking this function, wdyt?
```suggestion
/**
 * @deprecated Please use page object functions in `onboarding.flow.ts` and in `pages/onboarding/*`.
 */
```

Reply:
That's a good point, thanks ! I have updated for all onboarding deprecated functions in commit [59c7225](https://github.com/MetaMask/metamask-extension/pull/28202/commits/59c7225881b1ad68eb7ab1cd18adab72f64e97b2)